### PR TITLE
Vaccination site feature

### PR DESCRIPTION
**Pull request by jordan-caldwell**
## Summary
This PR provides the functionality to build out the new vaccination site. The functionality included is a new content type, view, and geolocation which required several modules all of which have been added as a dependency of the feature package. 

## Metadata
| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| `CHANGELOG` reflects changes? | yes
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | https://thinkoomph.jira.com/browse/RIGA-38

### CODE REVIEW

File: ecms_base/features/custom/ecms_vaccination_site/config/install/field.field.node.vaccination_site.field_vacsite_spec_instructions.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    full_html: '0'
+    plain_text: '0'
+id: node.vaccination_site.field_vacsite_spec_instructions
+field_name: field_vacsite_spec_instructions
+entity_type: node
+bundle: vaccination_site
+label: 'Special instructions'
+description: ''
+required: false
+translatable: false
```
Comment (ben-hamelin) [inline ecms_base/features/custom/ecms_vaccination_site/config/install/field.field.node.vaccination_site.field_vacsite_spec_instructions.yml:26]: Text field should be translatable

File: ecms_base/features/custom/ecms_vaccination_site/config/install/language.content_settings.taxonomy_term.vacsite_audiences_served.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.vacsite_audiences_served
+id: taxonomy_term.vacsite_audiences_served
+target_entity_type_id: taxonomy_term
+target_bundle: vacsite_audiences_served
+default_langcode: site_default
+language_alterable: false
```
Comment (ben-hamelin) [inline ecms_base/features/custom/ecms_vaccination_site/config/install/language.content_settings.taxonomy_term.vacsite_audiences_served.yml:10]: This should be true

File: ecms_base/features/custom/ecms_vaccination_site/config/install/language.content_settings.taxonomy_term.vacsite_indoor_outdoor.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.vacsite_indoor_outdoor
+id: taxonomy_term.vacsite_indoor_outdoor
+target_entity_type_id: taxonomy_term
+target_bundle: vacsite_indoor_outdoor
+default_langcode: site_default
+language_alterable: false
```
Comment (ben-hamelin) [inline ecms_base/features/custom/ecms_vaccination_site/config/install/language.content_settings.taxonomy_term.vacsite_indoor_outdoor.yml:10]: should be true

File: ecms_base/features/custom/ecms_vaccination_site/config/install/language.content_settings.taxonomy_term.vacsite_vaccine_type.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.vacsite_vaccine_type
+id: taxonomy_term.vacsite_vaccine_type
+target_entity_type_id: taxonomy_term
+target_bundle: vacsite_vaccine_type
+default_langcode: site_default
+language_alterable: false
```
Comment (ben-hamelin) [inline ecms_base/features/custom/ecms_vaccination_site/config/install/language.content_settings.taxonomy_term.vacsite_vaccine_type.yml:10]: should be true

**Review by ben-hamelin — CHANGES_REQUESTED** [review]
Just a few translation settings to review

**Comment by jordan-caldwell** [comment]
We're using google translator module for this site so it wont be using Drupal's translation engine

**Comment by ben-hamelin** [comment]
Ok all good then!
/approved

**Review by ben-hamelin — APPROVED** [review]
Yep!

**Pull request merged**